config/email.yaml.example

```diff
@@ -1,9 +1,5 @@
 # Outgoing email settings
 
 production:
-  delivery_method: :smtp
-  smtp_settings:
-    address: smtp.example.com
-    port: 25
-    domain: example.com
-    authentication: :none
+  email_delivery:
```

Comment:
I don't think this hash level is correct, `delivery_method` should be immediately under `production`. The initialiser fails with:

<pre>
/home/dcleal/.rvm/gems/ruby-2.3.0@foreman/gems/actionmailer-4.2.6/lib/action_mailer/base.rb:569:in `method_missing': undefined method `email_delivery=' for ActionMailer::Base:Class (NoMethodError)
        from /home/dcleal/code/foreman/foreman/config/initializers/email.rb:14:in `block in <top (required)>'
        from /home/dcleal/code/foreman/foreman/config/initializers/email.rb:12:in `each'
        from /home/dcleal/code/foreman/foreman/config/initializers/email.rb:12:in `<top (required)>'
</pre>